core/src/main/java/com/xatkit/core/recognition/IntentRecognitionProviderFactory.java

```diff
@@ -125,7 +140,14 @@ private static RecognitionMonitor getRecognitionMonitor(XatkitCore xatkitCore,
          */
         RecognitionMonitor monitor = null;
         if (configuration.isEnableRecognitionAnalytics()) {
-            monitor = new RecognitionMonitor(xatkitCore.getXatkitServer(), configuration.getBaseConfiguration());
+            if(configuration.getBaseConfiguration().getString(XATKIT_DATABASE_KEY, DEFAULT_DATABASE).equals(INFLUX_DATABASE)){
+                Log.info("Starting with influxDB");
+                monitor = new RecognitionMonitorInflux(xatkitCore.getXatkitServer(), configuration.getBaseConfiguration());
+            }
+            else { //Using mapDB, default value
```

Comment:
I have removed the comment here. It's already explained in the constants documentation.
Side note: do not put comment on the same line as the else construct, and use the following syntax for if/else statements:
```
if(condition) {
  // do something
} else {  <-- brackets on the same line.
  // do something else
}
```
(And yes I know I need to write this internal document about coding style, since it's not here I've fixed it for you 😉)